packages/docs/src/components/header/header.tsx

```diff
@@ -101,7 +101,9 @@ export const Header = component$(() => {
                 href="/examples/introduction/hello-world/"
                 class={{ active: pathname.startsWith('/examples') }}
```

Comment:
```suggestion
                class={{ active: pathname.startsWith('/examples') }}
                aria-label="Qwik playground" 
```

Reply:
I would stick with @erikras suggestion and remove directly the dash from the title, but it you still want to keep, please add the `aria-label` to the interactive element